src/assets/sass/protocol/components/_menu.scss

```diff
@@ -0,0 +1,327 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import '../includes/lib';
+
+@mixin highlighted {
+    &:after {
+        background: $color-black;
+        bottom: -1px;
+        content: '';
+        height: 1px;
+        left: 0;
+        position: absolute;
+        width: 100%;
+    }
+}
+
+/* -------------------------------------------------------------------------- */
+// Menu top navigation
+
+.mzp-c-menu {
+
+    h1, h2, h3, h4, h5, h6 {
+        @include open-sans;
+    }
+
+    @media #{$mq-md} {
+        z-index: 0;
+    }
+}
+
+.mzp-c-menu-category-list {
+    margin-bottom: $spacing-lg;
+
+    @media #{$mq-md} {
+        @include clearfix;
+        margin-bottom: 0;
+        padding-top: $spacing-lg;
+    }
+}
+
+.mzp-c-menu-category {
+
+    .mzp-c-menu-title {
+        @include open-sans;
+        @include text-body-md;
+        border-bottom: 1px solid $color-gray-30;
+        color: $color-black;
+        display: block;
+        font-weight: normal;
+        margin-bottom: 0;
+        min-height: 40px;
+        padding: $spacing-sm 0;
+        position: relative;
+        text-decoration: none;
+
+        &:hover,
+        &:active,
+        &:focus {
+            text-decoration: underline;
+        }
+
+        &:link:active {
+            background: none;
+        }
+    }
+
+    &.mzp-has-drop-down .mzp-c-menu-title {
+        cursor: default;
+        text-decoration: none;
+
+        &:before {
+            // base64 encoding of https://github.com/mozilla/protocol-assets/blob/master/icons/ui/expand-black.svg
+            background:  url('data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMTYiIGhlaWdodD0iMTYiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyI+PHBhdGggZD0iTTcgN1YxYTEgMSAwIDEgMSAyIDB2Nmg2YTEgMSAwIDEgMSAwIDJIOXY2YTEgMSAwIDEgMS0yIDBWOUgxYTEgMSAwIDEgMSAwLTJoNnoiIGZpbGw9IiMwMDAiIGZpbGwtcnVsZT0iZXZlbm9kZCIvPjwvc3ZnPgo=') top left no-repeat;
```

Comment:
My main line of reasoning here was so the icons are visible immediately when the menu is opened, and don’t require an http request. I think making them immediately obvious in the UI is perhaps more advantageous than avoiding the debt, but i’m open to other opinions.

Reply:
Yeah, that makes sense. And in the case of mobile I agree, the benefit of instantly available outweighs the cost of easily changeable (when it might never even change).